apps/app/codegen.ts

```diff
@@ -3,7 +3,15 @@ import { CodegenConfig } from '@graphql-codegen/cli';
 const config: CodegenConfig = {
   // schema: 'http://clinical-api.tau.health:8080/graphql',
   schema: 'http://clinical-api.boson.health/graphql',
-  documents: ['apps/app/src/**/Settings/**/*.tsx', 'apps/app/src/views/components/TicketModal.tsx'],
+  documents: [
+    'apps/app/src/**/Settings/**/*.tsx',
```

Comment:
just keep in mind, if we add files here there can only be service queries and mutations in these files, just fyi